MKOB/MKOB.py

```diff
@@ -25,19 +25,17 @@
 """
 
 """
-
-MKOB.pyw
+MKOB.pyw  (Why does this say MKOB.pyw instead of MKOB.py? I don't remember. @leskerr)
```

Comment:
Running a ".py" program with "python.exe" executes in a terminal window, which stays open while the program runs.  Changing too ".pyw" causes it to be run by pythonw.exe by default instead.  This suppresses the terminal window on startup, which may be more appropriate for a program that presents a GUI.

I think for the main "MKOB" program you probably want ".pyw".

Reply:
Thanks, @pwdirks. I changed the filename to MKOB.pyw. It seems to work either way with Windows. I'm not able to test it with Linux right now.